src/smt/smt_engine.cpp

```diff
@@ -1312,6 +1312,14 @@ void SmtEngine::setDefaults() {
         options::cbqi.set(true);
       }
     }
+    if (d_logic.isTheoryEnabled(THEORY_FP)
```

Comment:
Perhaps we should check that this is *pure* FP?  My concern is that many times we use the "ALL" logic and hence by default this would disable evaluation.

Reply:
Good point, will do.